### DISCUSSION TITLE
Show Member ID in the "Author" list

### DISCUSSION THREAD

**Discussion by shbchk** [ExpressionEngine/ExpressionEngine#4168] [Feature Requests]
### Suggest an idea for ExpressionEngine

When more than 1 person uses the same Screen Name, it becomes impossible to choose the author:

![image](https://github.com/ExpressionEngine/ExpressionEngine/assets/12234507/9fab6b08-8f05-4141-82b1-9298dcdad747)

**Comment by intoeetive** [comment]
@Yulyaswan does the component that we use here allow us to add additional text in smaller font? I think adding username could make sense

**Comment by Yulyaswan** [comment]
PR #4183

**Reply by bryannielsen** [reply]
@Yulyaswan it looks like this is adding the `author_id`, is it possible to insert the `username` like Yuri mentioned or `email` instead?  I think these might be a little more helpful in identifying the member unless you're concerned these values won't fit or look good in the select?

**Reply by Yulyaswan** [reply]
at the moment we have settled on the display ID

**Reply by intoeetive** [reply]
The member ID is the data we already have available; if we want to include the username, that would require significant rework of component
